admin/category_product_listing.php

```diff
@@ -679,30 +683,30 @@
                   <td class="hidden-sm hidden-xs"><!-- no model for categories --></td>
                   <td class="hidden-sm hidden-xs"><!-- no price for categories --></td>
                 <?php } ?>
-                <?php if ($search_result || SHOW_COUNTS_ADMIN == 'true') { ?>
+                <?php if ($search_result || SHOW_COUNTS_ADMIN === 'true') { ?>
```

Comment:
`$search_result` doesn't 'participate' in this section, since output is only rendered if `SHOW_COUNTS_ADMIN === 'true'` and can be removed from this statement.